extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -38,6 +40,17 @@ const TAG = 'amp-bind';
 const AMP_CSS_RE = /^(i?-)?amp(html)?-/;
 
 /**
+ * Tags under which bind should observe mutaitons to detect added/removed
+ * bindings.
+ * @type {!Object<string, boolean>}
+ * @private
+ */
+const DYNAMIC_TAGS = map({
+  'TEMPLATE': true,
+  'AMP-LIVE-LIST': true,
```

Comment:
Why don't we instead have these components emit something like

`this.dispatchCustomEvent('dom:changed-subtree', {root: element})`

And listen to that from here?

That would not require changing AMP bind as more such components come.